RELEASE.md

```diff
@@ -3,11 +3,14 @@
 This document describes the version policy and release process for the MongoDB Atlas SQL interface ODBC Driver.
 
 ## Versioning
-Versions will follow the [semantic versioning](https://semver.org/) system.  
+
+Versions will follow the [semantic versioning](https://semver.org/) system.
 The following guidelines will be used to determine when each version component will be updated:
 - **major**: backwards-breaking changes
 - **minor**: functionality added in a backwards compatible manner
 - **patch**: backwards compatible bug fixes
+- **pre-release**: The pre-release version
+- **libv**: to specify which version of `libmongosqltranslate` gets built with the driver during a release
 
 ## Release Process
 ### Pre-Release Tasks
```

Comment:
I think this is where we need more details.
Can you add something along these lines in the "Determine the correct version to be release" section.
And if something isn't quite right or not detailed enough, feel free to make it better :)

"
Updating the version of mongosqltranslate being bundled is reflected on the ODBC version as patch version bump (or pre-release version until the first public EAP version).
Also, add the version of libmongosqltranslate in the `fixversion` of the release JIRA.
"